std_msgs/String[] array